chia/wallet/wallet_state_manager.py

```diff
@@ -1149,11 +1151,14 @@ async def handle_clawback(
         if is_recipient is not None:
             spend_bundle = SpendBundle([coin_spend], G2Element())
             memos = compute_memos(spend_bundle)
+            spent_height: uint32 = (
+                uint32(0) if latest_coin_state.spent_height is None else uint32(latest_coin_state.spent_height)
+            )
```

Comment:
This does not address the coverage complaint.  https://github.com/Chia-Network/chia-blockchain/pull/15496/commits/9a2ed50a50e0724768cba358ad26c91c6902bdca  Probably either go back to the original, or take this slightly more explicit option.  Or maybe there's a good third option, I haven't looked too much.

```suggestion
            spent_height: uint32
            if latest_coin_state.spent_height is None:
                spent_height = uint32(0)
            else:
                spent_height = uint32(latest_coin_state.spent_height)
            )
```

Reply:
If they fix https://github.com/nedbat/coveragepy/issues/509 it would complain regardless.

Reply:
Continued at https://github.com/Chia-Network/chia-blockchain/pull/15496#pullrequestreview-1481700019.